src/cppwinrt.build.pre.props

```diff
@@ -9,7 +9,6 @@
   <Import Project="..\common.openconsole.props" Condition="'$(OpenConsoleDir)'==''" />
 
   <PropertyGroup Label="Globals">
-    <CppWinRTHeapEnforcement>AnyValueHereWillDisableTheOptOut</CppWinRTHeapEnforcement>

```

Comment:
whooooooooooaaa what

Reply:
Oh I wanted to commit this in a separate PR but forgot to split it off first.

I believe we added this to work around a bug in the XAML compiler (no surprises there I'm sure) because it used `std::shared_ptr` instead of `winrt::com_ptr` on a WinRT object. This bug seems to have been fixed a while ago so there doesn't seem to be any reason for us to still opt out of these safety checks.

Reply:
YESSS